msi/GitChooseEnvironmentDlg.wxs

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Hmm, I thought about it. I'll take another look at it. I thought it wouldn't work--but I could change my mind again once I re-evaluate. The UI (and the flow of it) has been under significant change.